test/functional/bundleadd/include/test.bats

```diff
@@ -13,8 +13,8 @@ setup() {
   create_manifest_tar 10 test-bundle
   chown_root "$DIR/web-dir/10/staged/$t1_hash"
   chown_root "$DIR/web-dir/10/staged/$t2_hash"
-  #create_fullfile_tar 10 $t1_hash
-  #create_fullfile_tar 10 $t2_hash
+  create_fullfile_tar 10 $t1_hash
+  create_fullfile_tar 10 $t2_hash
```

Comment:
Mind explaining why this is being changed?

Reply:
I had commented it out as a debug step and committed it on accident.